watcher/watcher.go

```diff
@@ -35,9 +37,24 @@ import (
 	"github.com/topfreegames/maestro/reporters"
 	"k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/metrics/pkg/apis/metrics/v1beta1"
 	metricsClient "k8s.io/metrics/pkg/client/clientset_generated/clientset"
 )
 
+func createRoomUsages(pods *v1.PodList, availableRooms map[string]bool) ([]*models.RoomUsage, map[string]int) {
+	var roomUsages []*models.RoomUsage
+	roomUsagesIdxMap := map[string]int{}
+	for i, pod := range pods.Items {
+		if availableRooms[pod.Name] == true {
```

Comment:
all elements in the map are true, it would be better to return an array instead of a map